### ISSUE TITLE
Coroutines?

### ISSUE THREAD

**Issue by P1NG2WIN** [ww-tech/roxie#19]
As I understand,  under the hood is used RXJava, are you planning to add the ability to use coroutines?

**Comment by michaelcarrano** [comment]
@P1NG2WIN yes, we are planning to add support for this in the future but I do not have a timeline for it. You are welcome to help contribute.

We would ideally also continue to provide RxJava support for those that still depend on it.

**Comment by dragossusi** [comment]
Check my fork/pull request #20
